### PR TITLE
ebs br: expose the volumebackup init pod ttl as a crd attribute

### LINKED ISSUE #5486: ebs br: add TTL configuration for init pod
## Feature Request

**Is your feature request related to a problem? Please describe:**

For some sensitive environment,  GC and PD scheduler needs to be done asap.

**Describe the feature you'd like:**

Expose the TTL for init pod in order to do fast failover.

**Describe alternatives you've considered:**

**Teachability, Documentation, Adoption, Migration Strategy:**

### PR DESCRIPTION
**Pull request by BornChanger**
### What problem does this PR solve?

Closes #5486 

### What is changed and how does it work?

Add a new attribute `spec.template.volumeBackupInitJobMaxActiveSeconds`  in volumebackup crd to specify the ttl for init pod, whose default value is 600 (seconds).

### Code changes

- [X] Has Go code change
- [ ] Has CI related scripts change

### Tests

- [ ] Unit test 
- [ ] E2E test 
- [X] Manual test 
img_v3_026o_57c7998c-96ce-42c0-97b8-9f6b86765f0g

- [ ] No code 

### Side effects

- [ ] Breaking backward compatibility 
- [ ] Other side effects: 

### Related changes

- [ ] Need to cherry-pick to the release branch 
- [ ] Need to update the documentation 

### Release Notes

Please refer to [Release Notes Language Style Guide](https://github.com/pingcap/tidb-operator/blob/master/docs/release-note-guide.md) before writing the release note.

```release-note

```

### CODE REVIEW

File: pkg/fedvolumebackup/backup/backup_manager.go [hunk 1]
```diff
@@ -504,22 +504,23 @@ func (bm *backupManager) buildBackupMember(volumeBackupName string, clusterMembe
...
+			Tolerations:                         backupTemplate.Tolerations,
+			ToolImage:                           backupTemplate.ToolImage,
+			ImagePullSecrets:                    backupTemplate.ImagePullSecrets,
+			ServiceAccount:                      backupTemplate.ServiceAccount,
+			CleanPolicy:                         backupTemplate.CleanPolicy,
+			PriorityClassName:                   backupTemplate.PriorityClassName,
+			CalcSizeLevel:                       backupTemplate.CalcSizeLevel,
+			AdditionalVolumes:                   backupTemplate.AdditionalVolumes,
+			AdditionalVolumeMounts:              backupTemplate.AdditionalVolumeMounts,
+			VolumeBackupInitJobMaxActiveSeconds: backupTemplate.VolumeBackupInitJobMaxActiveSeconds,
```
Comment (WangLe1321) [inline pkg/fedvolumebackup/backup/backup_manager.go:523]: You can set this value in the code block of `if initialize {}` below.

Comment (BornChanger) [reply pkg/fedvolumebackup/backup/backup_manager.go]: done

**Review by WangLe1321 — APPROVED** [review]

**Comment by nkg-** [comment]
Does this PR guarantee mutating operations (gc/schedulers/lighting/merge/etc) will be resumed after the init pod is killed (as per TTL). Or will it take more time for the respective leases to expire.

**Pull request merged**